src/app/detail/ui/ActivityLog.tsx

```diff
@@ -0,0 +1,35 @@
+import ActivityDescription from '@/components/activity/ActivityDescription'
+import { ActivityType } from '@/types/interfaces'
+
+import { Avatar, Box, Stack } from '@mui/material'
+import { BoldTypography, StyledTypography, VerticalLine, WrapperStack } from '@/app/detail/ui/styledComponent'
+import { getTimeDifference } from '@/utils/getTimeDifference'
+
+interface Prop {
+  log: any
+  isLast: boolean
+}
+
+export const ActivityLog = ({ log, isLast }: Prop) => {
+  const renderArgs =
+    log.activityType == ActivityType.WORKFLOWSTATE_UPDATE
+      ? [log.details?.prevWorkflowState?.type, log.details?.currentWorkflowState?.type]
+      : log.activityType == ActivityType.ASSIGN_TASK
+        ? [log.details?.assignedTo]
+        : []
+
+  return (
+    <WrapperStack>
+      <Stack direction="row" columnGap={4}>
+        {!isLast && <VerticalLine />}
+
+        <Avatar alt="user" src={log?.iconImageUrl || log?.avatarImageUrl} sx={{ width: '20px', height: '20px' }} />
+        <Stack direction="row" columnGap={1}>
+          <BoldTypography>{log.details.initiator}</BoldTypography>
+          <ActivityDescription activityType={log.activityType} args={renderArgs} />
+        </Stack>
+        <StyledTypography> {getTimeDifference(log.createdAt)}</StyledTypography>
+      </Stack>
+    </WrapperStack>
+  )
+}
```

Comment:
I don't think we need this component. Currently, we have `<TaskActivityComponent/>` which is a client component with `use client` directive on the top. Then `<ActivityLog/>` which is a child and also rendered in the client. Let's not render such static items in the client side. Please check out compositional pattern in next js docs: 
https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns